packages/core/graphql/codegen.yml

```diff
@@ -0,0 +1,10 @@
+schema: https://broker.staging.textile.dev/graphql
```

Comment:
@joewagner this might be useful information once you start exploring the GraphQL stuff